var/spack/repos/builtin/packages/scotch/package.py

```diff
@@ -6,15 +6,20 @@
 from spack import *
 
 
-class Scotch(Package):
+class Scotch(CMakePackage):
```

Comment:
Unfortunately this makes `cmake` a dependency even for older versions that don't use it... I think that's a fair tradeoff though.

Reply:
Looking forward to one day removing the old stuff